e2e/run

```diff
@@ -273,6 +273,8 @@ addons:
     enabled: true
 cloudWatchLogging:
   enabled: true
+ssmAgent:
```

Comment:
You are right. Wow, it was written at the first commit but have not been found until this. Thanks!